functions/global-setup.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
```

Comment:
Instead of disabling eslint, can you get this to work through typescript?
You'll probably need a custom tsconfig for the `functions/` folder - see `cypress/tsconfig.json` for an example.